inference-engine/src/mkldnn_plugin/nodes/mkldnn_bucketize_node.cpp

```diff
@@ -39,6 +39,9 @@ MKLDNNBucketizeNode::MKLDNNBucketizeNode(const std::shared_ptr<ngraph::Node>& op
 
     errorPrefix = "Bucketize layer with name '" + op->get_friendly_name() + "' ";
     const auto bucketsize = std::dynamic_pointer_cast<const ngraph::opset3::Bucketize>(op);
+    if (bucketsize == nullptr)
```

Comment:
1) I wat trying to understand why this missed nullptr check is detected only for few nodes.
As far as I understand all the nodes have the same pattern.

2) General comment, not particularly related to your changes:
Can we actually avoid this double "dynamic_pointer_cast" (one in constructor and one in isSupportedOperation) by casting first (before **isSupportedOperation** call) and then pass casted pointer (ngraph::opset3::Bucketize in this case) to isSupportedOperation).
Probably we could do it for all the nodes.

3) Probably it is better to have the same style for nullptr check (either (!pointer) or (pointer == nullptr))

Reply:
Or maybe we can go even further and perform the cast before passing to the constructor.
I mean we already know beforehand which ngraph OP corresponds to which mkldnn_node. Why do we want to postpone this dynamic cast till the node constructor?

Reply:
I suppose, these points are out of scope of current PR.
1. Here are fixes of issues highlighted by tool. I don't know, why it does not show/detect other cases.
2. As a case we can compare type_info in fn isSupportedOperation. Perhaps it could be a little bit faster.
3. We don't have such rules.